clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -285,12 +286,12 @@ def err_drv_omp_host_target_not_supported : Error<
 def err_drv_expecting_fopenmp_with_fopenmp_targets : Error<
   "The option -fopenmp-targets must be used in conjunction with a -fopenmp option compatible with offloading, please use -fopenmp=libomp or -fopenmp=libiomp5.">;
 def err_drv_expecting_fsycl_with_sycl_opt : Error<
-  "The option %0 must be used in conjunction with -fsycl to enable offloading.">;
+  "'%0' must be used in conjunction with '-fsycl' to enable offloading">;
 def err_drv_fsycl_with_c_type : Error<
-  "The option '%0' must not be used in conjunction with '-fsycl', which expects C++ source.">;
+  "'%0' must not be used in conjunction with '-fsycl', which expects C++ source">;
 def warn_drv_sycl_offload_target_duplicate : Warning<
-  "The SYCL offloading target '%0' is similar to target '%1' already specified - will be ignored.">,
-  InGroup<SyclTarget>;
+  "SYCL offloading target '%0' is similar to target '%1' already specified; "
+  "will be ignored">, InGroup<SyclTarget>;
```

Comment:
Should we update line 300 as well?

Reply:
Not as part of this review -- that exists in community as well.

Reply:
I'm asking because, I don't understand the reason to change " - will be ignored" to "; will be ignored".
I'm using the same wording in my pull request - https://github.com/intel/llvm/pull/3929.
Which version should I use?

Reply:
Common style is to use `;` and not `-`. I'm changing upstream to use `;` as well (https://reviews.llvm.org/D107402), so I'd recommend doing the same in #3929.

Reply:
Thanks for clarification. I'll apply this suggestion to #3929.